common/scala/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/CosmosDBArtifactStore.scala

```diff
@@ -445,7 +462,8 @@ class CosmosDBArtifactStore[DocumentAbstraction <: DocumentSerializer](protected
       Seq(
         (cid, Some(JsString(escapeId(json.fields(_id).convertTo[String])))),
         (etag, json.fields.get(_rev)),
-        (computed, computedOpt))
+        (computed, computedOpt),
+        (clusterId, clusterIdValue))
```

Comment:
Minor: It may be worth noting (comment in the cluster id config?) that clusterId may change in the case where doc is created from one cluster, and edited from another, when both clusters use the same collection.

Reply:
Documented it now as part of scaladoc for `CosmosDBConstants`